euth/comments/apps.py

```diff
@@ -1,5 +0,0 @@
-from django.apps import AppConfig
-
-
-class CommentsApiConfig(AppConfig):
-    name = 'comments'
```

Comment:
Because it seemed to that it wasn't used. As far as I see apps.py only is used if `__init.py__` does contains `default_app_config =`